diagnostics-extension/webpack.config.js

```diff
@@ -18,4 +18,7 @@ module.exports = {
     filename: "sw.js",
     path: path.resolve(__dirname, "build"),
   },
+  resolve: {
+    extensions: [".ts"],
+  },
```

Comment:
Can you ellaborate on what this means?

Reply:
This allows webpack to find the appropriate imports from other files.

Eg:

```
import * as fakeData from './fake_telemetry.data';
```
At the end we don't include `.ts` extension here (a convention)

By default webpack is configured to append `.js` at the end if no extension is found. So we've to tell webpack to replace the blanks with `.ts` during compilation.

Reply:
I missed this line out while setting up webpack because earlier we didn't have any imports. As soon as I added some, the build started failing because it couldn't find `./fake_telemetry.data.js` at compilation time.